dlt/common/storages/load_package.py

```diff
@@ -514,3 +621,59 @@ def filter_jobs_for_table(
         all_jobs: Iterable[LoadJobInfo], table_name: str
     ) -> Sequence[LoadJobInfo]:
         return [job for job in all_jobs if job.job_file_info.table_name == table_name]
+
+
+@configspec
+class LoadPackageStateInjectableContext(ContainerInjectableContext):
+    storage: PackageStorage
+    load_id: str
+    can_create_default: ClassVar[bool] = False
+    global_affinity: ClassVar[bool] = True
```

Comment:
it should be false, no? you can have many pipelines running at once and each of them will maintain separate context. the only context that has global affitity are config providers

Reply:
- This is a left over of yesterdays battle i think :D